stacks/java-websphereliberty-gradle/devfile.yaml

```diff
@@ -18,9 +18,8 @@
 schemaVersion: 2.1.0
 metadata:
   name: java-websphereliberty-gradle
-  version: 0.4.0
-  displayName: 'WebSphere Liberty Gradle'
-  description: Java application Gradle-built stack using the WebSphere Liberty runtime
+  displayName: WebSphere Liberty Gradle
+  description: Java application based on OpenJ 9 Java 11 and Gradle, using the WebSphere Liberty runtime 22.0.0.1
```

Comment:
Similar to the java-openliberty-gradle. Mentioning both OpenJDK 9 and Java 11 is in conflict.